ARM.CMSIS.pdsc

```diff
@@ -2174,16 +1308,290 @@ and 8-bit Java bytecodes in Jazelle state.
       <require Cclass="Device" Cgroup="Startup"/>
     </condition>
 
+    <condition id="RTOS RTX ARM ARMv6-M">
+        <description>Arm Compiler for Armv6-M architecture</description>
```

Comment:
add "(little endian)"

Reply:
Okay, I have blindly taken little endian as the default.

Reply:
Ack, I'll add this.